pkg/cluster/cluster.go

```diff
@@ -27,7 +27,7 @@ import (
 	"time"
 
 	sdk "github.com/openshift-online/ocm-sdk-go"
-	amsv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
```

Comment:
was this intentional ? In the whoami package we are also using amsv1.

Reply:
I changed these to keep is consistent with CS (they have used cmv1 instead of cmsv1), but we can revert this back